scout/build/case.py

```diff
@@ -34,6 +34,17 @@ def build_phenotype(phenotype_id, adapter):
     return phenotype
 
 
+def _populate_pipeline_info(case_obj, case_data):
```

Comment:
Perhaps also add the `reference_info` then if we decided to load them systematically. They complement each other well for MIP?